provider/filestore.go

```diff
@@ -266,13 +267,30 @@ func (s3 *S3FileStore) PathWithPrefix(path string, remote bool) string {
 }
 
 type GCSFileStore struct {
+	Bucket string
+	Path   string
 	genericFileStore
 }
 
+func (gs *GCSFileStore) PathWithPrefix(path string, remote bool) string {
+	gsPrefix := "gs://"
+	noGSPrefix := path[:len(gsPrefix)] != gsPrefix
```

Comment:
https://pkg.go.dev/strings#HasPrefix

I know its the same thing but way more readable to use strings.HasPrefix